notebooks/bigQuery.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4d79e85b",
+   "metadata": {},
+   "source": [
+    "## Big Query SQL queries used in this project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df71635a",
+   "metadata": {},
+   "source": [
+    "Here we're collecting lists of page paths and content ids, for which there have been more than 5 page hits in the last 2 weeks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc65c551",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import packages\n",
+    "import pandas as pd\n",
+    "import pandas_gbq\n",
+    "\n",
+    "# extract data from BigQuery\n",
```

Comment:
We need to make sure users are authenticated when querying BigQuery, see (partly) Alessia's document: https://docs.google.com/document/d/1H-S0wesMFxreqbYv4g_3eegxsMnL5a_qmGRbMf1Xp60/edit#. We are using pandas_gbq to access and authenticate users. We need to use a service account credentials file to do this if we are putting this on GitHub (I'm not sure if you have one of those yet (its a json file) If not, add a message on the data labs slack channel and someone will get you one)

So we need to add something like: 
  
```
from google.oauth2 
import service_account 
import pandas_gbq
credentials = service_account.Credentials.from_service_account_file(GOOGLE_APPLICATION_CREDENTIALS,)
```

And then change the 'create dataframe' bit to: 
 
`page_path = pandas_gbq.read_gbq(sql, project_id=project_id, credentials=credentials)
` 

Therefore, we need to add the json file path key to our secrets file, for example, mine would be: 
 
`export GOOGLE_APPLICATION_CREDENTIALS="/Users/alicestephenson/gds/secrets/govuk-bigquery-analytics-credentials.json"
` 

We also need to have this somewhere in the script then:
 
`GOOGLE_APPLICATION_CREDENTIALS = os.getenv("GOOGLE_APPLICATION_CREDENTIALS") `